ext/SparseDiffToolsZygote.jl

```diff
@@ -1,3 +1,24 @@
+module SparseDiffToolsZygote
+
+import Zygote
+
+using SparseDiffTools
+using SparseDiffTools: DeviVecTag, FwdModeAutoDiffVecProd
```

Comment:
Needs to look like https://github.com/JuliaArrays/ArrayInterface.jl/blob/master/ext/ArrayInterfaceGPUArraysCoreExt.jl